pkg/plugins/resources/k8s/store.go

```diff
@@ -93,6 +94,23 @@ func (s *KubernetesStore) Delete(ctx context.Context, r core_model.Resource, fs
 	}
 	return nil
 }
+func (s *KubernetesStore) DeleteMany(ctx context.Context, fs ...store.DeleteManyOptionsFunc) error {
```

Comment:
I don't think it's a good idea to remove resources without taking into account their type